recipe/meta.yaml

```diff
@@ -2,20 +2,26 @@ package:
     name: mingwpy
     version: 0.1.0
 
-source:                                                                                                # [win]
+source:
```

Comment:
> It got clobbered in the merge. Probably due to the excess spaces here and my bad eyes.

No worries. I just noticed that I failed to explain the importance of this otherwise inane tweak last time. So, I felt the need to clarify and stress the importance of it.

> Maybe we should fix the script...

The script is just calling `conda-build` at this point. So, to repharse, you're saying maybe we should fix `conda-build`. 😉 

Sure, we should add an issue upstream. In the meantime, this is the easy fix to get things back up and running.

> ...instead of adding that to all feedstock that might skip stuff and create an empty field...

Sadly we did this once before. Hence this line.